### PR TITLE
More compara tests

### PR DESCRIPTION
**Pull request by muffato**
Yeah !
They now cover all the compara endpoints

### CODE REVIEW

**Comment by muffato** [comment]
On farm3 the test-suite failed in some variation endpoints. I think it's partly because I didn't have the C thing installed. I hope I haven't messed up any of their endpoints

**Comment by muffato** [comment]
Ah, the test dbs are on the e85 schema but Travis is using master which already has a schema changes for e86. I need to patch the db

**Comment by muffato** [comment]
Funnily, patch_test_database.pl adds "DEFAULT CHARSET=latin1" to every table :)

**Comment by muffato** [comment]
info.t will fail because it expects the compara db to be in the same schema as the core API

**Comment by magaliruffier** [comment]
can you please rebase your fork against master to ensure we have the latest hstlib fixes in?

**Comment by muffato** [comment]
I can see that you've already run the schema_patcher. I don't need 0511efc any more

**Comment by magaliruffier** [comment]
I think that's good to go

File: t/gene_tree.t [hunk 1]
```diff
@@ -0,0 +1,558 @@
+# Copyright [2016] EMBL-European Bioinformatics Institute
+# Copyright [2016] EMBL-European Bioinformatics Institute
```
Comment (nerdstrike) [inline t/gene_tree.t]: Mangled copyright

**Comment by muffato** [comment]
Thank you !

**Pull request merged**